packages/android_intent_plus/android/src/main/java/dev/fluttercommunity/plus/androidintent/MethodCallHandlerImpl.java

```diff
@@ -91,7 +92,14 @@ public void onMethodCall(@NonNull MethodCall call, @NonNull Result result) {
         sender.buildIntent(
             action, flags, category, data, arguments, packageName, componentName, type);
 
-    if ("launch".equalsIgnoreCase(call.method)) {
+    if ("parseAndLaunch".equalsIgnoreCase(call.method)) {
+      try {
+        intent = sender.parse(call.argument("uri"));
+        sender.send(intent);
+      } catch (URISyntaxException e) {
+        result.error("WRONG_URI", "Uri's syntax is wrong", null);
```

Comment:
add the exception message to the `result.error` so users will be able to see it, i.e. the error details: https://api.flutter.dev/javadoc/io/flutter/plugin/common/MethodChannel.Result.html#error(java.lang.String,java.lang.String,java.lang.Object)

Reply:
I have put error code and error message which is enough to get feedback, error details is optional. But anyway I have added the message for both.